cmd/limactl/copy.go

```diff
@@ -128,14 +154,28 @@ func copyAction(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
+
 	sshArgs := sshutil.SSHArgsFromOpts(sshOpts)
 
-	sshCmd := exec.Command(arg0, append(sshArgs, scpArgs...)...)
+	sshCmd := exec.Command(arg0, createArgs(sshArgs, copyToolArgs, defaultTool)...)
 	sshCmd.Stdin = cmd.InOrStdin()
 	sshCmd.Stdout = cmd.OutOrStdout()
 	sshCmd.Stderr = cmd.ErrOrStderr()
```

Comment:
We run rsync via ssh? it does not make sense.

Can you share the complete the scp and rsync commands that you want to run?

The commands can be tested from the shell without lima, using the instance ip. limactl just make this easier for the user by building the right command and running it transparently.

Reply:
for scp 

```
scp -F ~/.lima/deb/ssh.config "myfile.txt" olalekanodukoya@lima-deb:/dump
myfile.txt                                            100%    7     2.1KB/s   00:00
```